JSDemos/Demos/TabPanel/SortableClosableTabs/React/App.js

```diff
@@ -36,40 +35,41 @@ function App() {
     if (index >= newEmployees.length && index > 0) {
       setSelectedItem(newEmployees[index - 1]);
     }
-  }
+  }, [employees, setEmployees, setSelectedItem]);
 
-  function renderTitle(data) {
+  const renderTitle = React.useCallback((data) => {
     function closeHandler() {
```

Comment:
is we pass an arrow function to `onClick` why we need this function? Cannot we just pass `() => { closeButtonHandler(data); }`

Reply:
It is a rudiment from previous version. Thanks.
https://github.com/DevExpress/devextreme-demos/pull/1916/commits/15f8186a1bee53602904eb8267b2fcd2f0b6b1f2